desktop/release-notes.md

```diff
@@ -24,6 +24,75 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/general.md/#releases)
 
+## Docker Desktop 4.12.0
+2022-09-01
+
+> Download Docker Desktop
+>
+> {% include desktop-install.html %}
+
+## New
```

Comment:
I think some of these can be redistributed to the Bug fixes and minor changes sections. I'd suggest moving:

- Minimum version to install or update Docker Desktop on macOS is now 10.15
-Improved overall performance issues caused by calculating disk size. Related to [docker/for-win#9401](https://github.com/docker/for-win/issues/9401).
-  Enable ComposeV2 after factory reset
- Upgrade kernel to 5.10.124
- Prevents users on ARM macs without Rosetta installed from switching back to ComposeV1 (only has intel binaries).
- - Descending default sort order for volumes size and created column, containers started column
- Defined sort order for container/compose state column running > some running > paused > some paused > exited > some exited > created
- Re-organized container row actions
- - Runs every command on the Quickstart guide immediately.
- Updated the tooltip design.

Reply:
RIght, thanks! I have moved these